lib/ravelin.rb

```diff
@@ -1,5 +1,39 @@
-require "ravelin/version"
+require 'date'
+require 'json'
+require 'faraday'
+require 'faraday_middleware'
+
+require 'ravelin/version'
+
+require 'ravelin/errors/api_error'
+require 'ravelin/errors/invalid_request_error'
+require 'ravelin/errors/authentication_error'
+require 'ravelin/errors/rate_limit_error'
+
+require 'ravelin/ravelin_object'
+require 'ravelin/chargeback'
+require 'ravelin/customer'
+require 'ravelin/device'
+require 'ravelin/item'
+require 'ravelin/location'
+require 'ravelin/order'
+require 'ravelin/payment_method'
+require 'ravelin/pre_transaction'
+require 'ravelin/transaction'
+
+require 'ravelin/event'
+require 'ravelin/response'
+require 'ravelin/client'
 
 module Ravelin
-  # Your code goes here...
+  @faraday_adapter = Faraday.default_adapter
```

Comment:
Faraday.default_adapter does not open persistent connection. You might want to change it to `net_http_persistent` adapter.